sysid/src/main/native/cpp/analysis/AnalysisManager.cpp

```diff
@@ -99,10 +99,10 @@ void AnalysisManager::PrepareGeneralData() {
   wpi::StringMap<std::vector<PreparedData>> preparedData;
 
   // Store the raw data columns.
-  static constexpr size_t kTimeCol = 0;
-  static constexpr size_t kVoltageCol = 1;
-  static constexpr size_t kPosCol = 2;
-  static constexpr size_t kVelCol = 3;
+  constexpr size_t kTimeCol = 0;
```

Comment:
Isn’t static fine for the .cpp file ones?

Reply:
There's no reason to declare them static in this context because they're constexpr.